### ISSUE TITLE
Feature: Create an option to invoice for all remaining funds in a collective

### ISSUE THREAD

**Issue by RichardLitt** [opencollective/opencollective#6488]
## Who is your user?

Often, an admin of a collective will want to empty that collective - either permanently, or to just use all of the remaining funds for now, in expectation of new funds. Their main way of doing this is to submit an invoice for the amount of funds left in the collective. 
However, they don't always know that each invoice comes with hidden expenses - from PayPal, from Wise, or from banks - with the transfer, and they have no way of _a priori_ knowing what that expense will come to. This makes it hard for them to adequately invoice for the remaining funds. 

Another user is us - the Fiscal Host who is paying out invoices. We frequently have to tell users to adjust their invoice to a level below the threshold of fees, so that they can be paid. This means they need to go edit their invoice and number, which adds time and admin. 

## What are they trying to achieve?

Users: Just empty the collective into their own bank account.

Admins: Pay out a user on the first go, without having to tell them to edit it. 

## Suggested fix

Have a way of submitting an invoice that uses all of the funds in the repo, but leaves just enough off of the invoice to cover fees to their payout mechanism. Programmatically figure out what the fees will be before they file an invoice for all of the fees.

**Comment by RichardLitt** [comment]
Note: This may not be possible with PayPal. 

Also, cf. https://docs.opencollective.com/help/expenses-and-getting-paid/expenses#what-are-the-approximate-fees-charged-by-each-payment-processor-credit-card-paypal-bank-transfer

**Comment by Betree** [comment]
In https://github.com/opencollective/opencollective/issues/1073 we've implemented a checkbox in the pay expense modal to solve this particular issue:

![image](https://user-images.githubusercontent.com/1556356/222069349-e20af266-3241-4bd1-800f-40963463813e.png)

When checked, the fees are deducted from the payment rather than "on top", which allows you to easily pay the full collective balance. And indeed, this is [not supported by PayPal](https://github.com/opencollective/opencollective/issues/5070). I searched and it looked like we've never documented this, I'll look into that.

Does that answer the need surfaced here?

**Comment by RichardLitt** [comment]
Interesting. I don't think this answers that. We pay the fees, and we don't want the user to take on that obligation.

I think we need to know before submitting what the fees will be for a given payment processor, which may not be possible programmatically.

**Comment by Betree** [comment]
> Interesting. I don't think this answers that. We pay the fees, and we don't want the user to take on that obligation.

Well, in the end it's the same result: the user will receive the collective balance minus the payment processor fees. But it does need to be clearly explained to them beforehand.

**Comment by RichardLitt** [comment]
Fair point. And yes, I think it would need to be messaged differently. 

Right now, though, does this work for all types of payouts?

**Comment by Betree** [comment]
> Right now, though, does this work for all types of payouts?

No, unfortunately, manual payouts are not supported yet and PayPal cannot support this: https://github.com/opencollective/opencollective/issues/5070
